python/cuvs_bench/cuvs_bench/run/__main__.py

```diff
@@ -80,6 +80,14 @@
     "RAPIDS_DATASET_ROOT_DIR if defined, otherwise a datasets "
     "subdirectory from the calling directory.",
 )
+@click.option(
+    "--executable-dir",
+    default=None,
+    show_default=True,
+    prompt="Enter the name of the folder that contains the executables",
+    help="Path to executable folder, by default we will look in "
+    "$CUVS_HOME/cpp/build/release and $CONDA_PREFIX/bin/ann.",
```

Comment:
Is it `build/release` or just `build`? Also do we look in `build` first then conda prefix?